### ISSUE TITLE
Cannot compile after updating from 3.2.0 to 3.3.2

### ISSUE THREAD

**Issue by md-griddy** [rwbutler/Connectivity#31]
Running into an issue where the the file directory is messed up now and cannot find the correct file paths for the following:

`error: no such file or directory: /Users/travis/build/gogriddy/app_ios/Pods/Connectivity/Connectivity/Classes/Response Validation/Validators/ConnectivityResponseContainsStringValidator.swift
[02:13:13]: ▸ ❌  error: no such file or directory: /Users/travis/build/gogriddy/app_ios/Pods/Connectivity/Connectivity/Classes/Response Validation/Validators/ConnectivityResponseRegExValidator.swift
[02:13:13]: ▸ ❌  error: no such file or directory: /Users/travis/build/gogriddy/app_ios/Pods/Connectivity/Connectivity/Classes/Response Validation/Validators/ConnectivityResponseStringEqualityValidator.swift
[02:13:13]: ▸ ❌  error: no such file or directory: /Users/travis/build/gogriddy/app_ios/Pods/Connectivity/Connectivity/Classes/Response Validation/Validators/ConnectivityResponseStringValidator.swift
[02:13:13]: ▸ ❌  error: no such file or directory: /Users/travis/build/gogriddy/app_ios/Pods/Connectivity/Connectivity/Classes/Response Validation/ConnectivityResponseValidationMode.swift
[02:13:13]: ▸ ❌  error: no such file or directory: /Users/travis/build/gogriddy/app_ios/Pods/Connectivity/Connectivity/Classes/Response Validation/ConnectivityResponseValidator.swift
[02:13:13]: ▸ ❌  error: no such file or directory: /Users/travis/build/gogriddy/app_ios/Pods/Connectivity/Connectivity/Classes/Response Validation/Factory/Factory.swift
[02:13:13]: ▸ ❌  error: no such file or directory: /Users/travis/build/gogriddy/app_ios/Pods/Connectivity/Connectivity/Classes/Response Validation/Factory/ResponseValidatorFactory.swift`

3.2.0 worked fine. The changes you guys made to renaming or deleting files caused these issues.

**Comment by rwbutler** [comment]
I've integrated the pod into a new project which seems to work fine at the moment - one thing I've noticed is there is one too many 'Connectivity' in the paths above. Did this issue crop up after running `pod update`?

**Comment by md-griddy** [comment]
Yes, it arose after I ran pod update to go to the latest version (at the time). It looks like in your change history that you guys have renamed some files and also altered some filepaths which makes Xcodebuild confused as it's still trying to reference to old file locations for some reason. I have tried everything from removing Connectivity library completely (removing it from the Podfile and running `pod install -no-repo-update` to removing the pertinent files itself within the Bulid Phases and re-adding them manually. They still are referencing the old file paths and I can get it to recognize the new file paths or filenames that your team has renamed them to.
